pkg/kubelet/util/manager/cache_based_manager.go

```diff
@@ -230,6 +235,10 @@ func (c *cacheBasedManager) RegisterPod(pod *v1.Pod) {
 	prev = c.registeredPods[key]
 	c.registeredPods[key] = pod
 	if prev != nil {
+		if prev.Namespace != pod.Namespace || prev.Name != pod.Name {
+			panic(fmt.Errorf(c.groupResource.String()+".236: mismatch in key/obj: %#v, %#v", key, prev))
```

Comment:
`.236`?

Reply:
Oh, line number

Reply:
yeah, dummy commit with logging, not for merge